automerge/src/columnar.rs

```diff
@@ -592,6 +592,9 @@ impl ValEncoder {
                 let len = (*n).encode(&mut self.raw).unwrap();
                 self.len.append_value(len << 4 | VALUE_TYPE_IEEE754);
             }
+            ScalarValue::Unknown { type_code, bytes } => {
+                panic!("unknown value")
```

Comment:
Is there a plan for encoding unknown values in the future?

Reply:
This code would always have panicked in the case of unknown values so this isn't strictly a downgrade. The new code does handle this case and now that we've removed the old encoding/decoding code (i.e. this code) it doesn't matter anyway.